src/postcodeInfoClient.ts

```diff
@@ -0,0 +1,78 @@
+import * as request from 'request-promise-native'
+import { Address } from './address'
+import { Point } from './point'
+import { PostcodeInfoResponse } from './postcodeInfoResponse'
+
+export class PostcodeInfoClient {
+  constructor (private readonly apiToken: string,
+               private readonly apiUrl: string = 'https://postcodeinfo.service.justice.gov.uk') {
```

Comment:
should we maybe set this using ansible? do they have a test platform or is it just their prod?

Reply:
This follows the pattern set by the existing libraries, the url is public and hardcoded in the libraries with the option to override it